setup.py

```diff
@@ -67,17 +96,42 @@ def run(self):
         call('make -C docs/ clean', shell=True)
 
 
-class TestCoverage(SimpleCommand):
+class Test(TestCommand):
+    """Run all tests."""
+
+    description = 'run tests and display results'
+
+    def get_args(self):
+        """Return args to be used in test command."""
+        markers = self.size
+        if markers == "small":
+            sizes = ['medium', 'large']
+            markers = "".join(['not %s and ' % size for size in sizes])[:-5]
```

Comment:
no, no, this loop was created because before the medium and large were defined only in parent class, but I'll create a static variable for this. Thanks @hdiogenes